### ISSUE TITLE
Updated to Meteor 1.3 - meteorInstall is not defined

### ISSUE THREAD

**Issue by applebya** [meteor-useraccounts/flow-routing#35]
Hi guys,

Having some trouble with this package, having updated to Meteor 1.3. Getting the following error:

```
/Users/Andrew/.meteor/packages/meteor-tool/.1.3.1.co8qz4++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:267

ReferenceError: meteorInstall is not defined
at packages/useraccounts_flow-routing.js:15:15
at packages/useraccounts_flow-routing.js:323:3
at /Users/Andrew/App/.meteor/local/build/programs/server/boot.js:283:10
```

I checked out the file, and meteorInstall isn't actually defined.

Any help would be really appreciated. Had to downgrade back to 1.2 in the meantime :(

Thanks - - Andrew

**Comment by joncursi** [comment]
Ran into the same problem after installing this package in my Meteor 1.3.1 app. Had to drop the package altogether.

**Comment by gwendall** [comment]
Fixed, I just opened a PR

**Comment by applebya** [comment]
Sweet, thanks @gwendall

**Comment by gczh** [comment]
I'm facing the same issue as well! Has anyone managed to find a fix?

**Comment by luiseduardobrito** [comment]
I'm having this issue with Meteor 1.4, is it supported?

**Comment by Ingosmar89219** [comment]
im having this issue with Meteor 1.4.2, is it supported?

**Comment by herlon214** [comment]
im having this issue with Meteor 1.4.2.3, is it supported?

**Comment by ziarno** [comment]
Why is this closed? I'm having the same issue on Meteor 1.4.4.2

**Comment by erwzqsdsdsf** [comment]
I have same in 1.8.1
